packages/LUIS/bin/luis.js

```diff
@@ -105,6 +105,8 @@ async function runProgram() {
     args.region = args.region || serviceIn.region || config.region || "westus";
     args.cloud = args.cloud || serviceIn.cloud || config.cloud;
     args.customHeaders = { "accept-language": "en-US" };
+    args.endpoint = args.endpoint || serviceIn.endpoint || config.endpoint;
+    args.authoringEndpoint = args.authoringEndpoint || serviceIn.authoringEndpoint || config.authoringEndpoint || `https://${args.region}.api.cognitive.microsoft.com/luis/api/v2.0/`;
```

Comment:
>authoringEndpoint [](start = 9, length = 17)

no need for authoring endpoint, combo of endpoint + cloud is sufficient.
"endpoint":"westeurope",
"cloud":"eu"